docs/source/guide/pt-1-intro.md

```diff
@@ -50,56 +52,84 @@ the circuit on a noisy simulator, and returns the probability of the ground
+by applying a rotation around the X-axis (Rx gate) to each output of any 2-qubit gate.
+
+This noise model is well-suited to highlight the effect of Pauli Twirling,
+which is a technique that turns coherent noise into incoherent Pauli channels.
+
+For the sake of this example, we define the noise level as the angle of the Rx rotation.
+
 ```{code-cell} ipython3
-import numpy as np
-from cirq import DensityMatrixSimulator, amplitude_damp
-from mitiq.interface import convert_to_mitiq
+from cirq import CircuitOperation, CXPowGate, CZPowGate, DensityMatrixSimulator, Rx
+from cirq.devices.noise_model import GateSubstitutionNoiseModel
 
-def execute(circuit, noise_level=0.1):
-    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit
-    executed with amplitude damping noise.
+def get_noise_model(x_rotation: float) -> GateSubstitutionNoiseModel:
+    """Substitute each C-Z and C-NOT gate in the circuit 
+    with the gate itself followed by an Rx rotation on the output qubits
     """
-    # Replace with code based on your frontend and backend.
-    mitiq_circuit, _ = convert_to_mitiq(circuit)
-    noisy_circuit = mitiq_circuit.with_noise(amplitude_damp(gamma=noise_level))
-    rho = DensityMatrixSimulator().simulate(noisy_circuit).final_density_matrix
-    return rho[0, 0].real
+    def noisy_c_gate(op):
+        if isinstance(op.gate, (CZPowGate, CXPowGate)):
+            return CircuitOperation(
+                Circuit(
+                    op.gate.on(*op.qubits), 
+                    Rx(rads=x_rotation).on_each(op.qubits),
+                ).freeze())
+        return op
+
+    return GateSubstitutionNoiseModel(noisy_c_gate)
+
+def execute(circuit: Circuit, noise_level: float):
+    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit"""
+    return (
+        DensityMatrixSimulator(noise=get_noise_model(x_rotation=noise_level))
+        .simulate(circuit)
+        .final_density_matrix[0, 0]
+        .real
+    )
 ```
 
 The [executor](executors.md) can be used to evaluate noisy (unmitigated)
 expectation values.
 
 ```{code-cell} ipython3
-# Compute the expectation value of the |0><0| observable.
-noisy_value = execute(circuit)
+# Set the intensity of the noise
+NOISE_LEVEL = 0.1
+
+# Compute the expectation value of the |0><0| observable
+# in both the noiseless and the noisy setup
 ideal_value = execute(circuit, noise_level=0.0)
-print(f"Error without mitigation: {abs(ideal_value - noisy_value) :.3}")
+noisy_value = execute(circuit, noise_level=NOISE_LEVEL)
+
+print(f"Error without twirling: {abs(ideal_value - noisy_value) :.3}")
 ```
 
 ## Apply PT
-Pauli Twirling can be easily implemented with the function
-{func}`.pauli_twirl_circuit()`.
+PT can be applied by first generating twirled variants of the circuit with the function
+{func}`.generate_pauli_twirl_variants` from the `mitiq.pt` module, 
+and then averaging over the results obtained by executing those variants.
```

Comment:
You should make it clear here that you are using the default value of `num_circuits = 10` in `generate_pauli_twirl_variants`.The more variants are generated and averaged over, the better the results are.